README.md

```diff
@@ -1 +1,45 @@
-## Hello Microverse
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+>A simple project to display Hello Microverse with my favorite color.
+
+## Built With
+
+- HTML5
+- CSS#
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- All you need is a computer
+### Setup
+- Clone the project from this repository into your local machine.
+- `cd` into the root directory and run the project using a live server or by right clicking on the index.html and opening it with your favorite browser.
+
+## Author
+
+👤 **Emanuel Okello**
+
+- GitHub: [oxenprogrammer](https://github.com/oxenprogrammer)
+- Twitter: [@ox_emmy](https://twitter.com/ox_emmy)
+- LinkedIn: [Emanuel Okello](https://www.linkedin.com/in/emanuel-okello/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/oxenprogrammer/curriculum-cr-training-contact-form/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Your README is descriptive enough for this kind of project however, we need to make it as professional as possible. The link to the issues page here is for another repository. Please fix it because we need to project a professional outlook.
- [ ] The MIT license link too is broken. Public repositories on GitHub are often used to share open source software. You'll need to license your repository appropriately so that others are free to use, change, and distribute the software or otherwise.